main.go

```diff
@@ -73,7 +74,9 @@ func main() {
 	flag.BoolVar(&testRunLogs, "test-logs", false, "Tests whether log collection works (does not test privilege dropping for local log collection, use --test for that)")
 	flag.BoolVar(&testExplain, "test-explain", false, "Tests whether EXPLAIN collection works by issuing a dummy query (ensure log collection works first)")
 	flag.StringVar(&testSection, "test-section", "", "Tests a particular section of the config file, i.e. a specific server, and ignores all other config sections")
-	flag.StringVar(&generateStatsHelperSql, "generate-stats-helper-sql", "", "Generates a SQL script for the given server (name of section in the config file, or \"default\" for env variables), that can be run with \"psql -f\" for installing the collector stats helpers on all configured databases")
+	flag.StringVar(&generateHelperSql, "generate-stats-helper-sql", "", "Deprecated alias for --generate-helper-sql")
```

Comment:
Do we really need to deprecate this? Why not remove it?

Reply:
Somewhat related to that, can you explain a bit more around here?

Reading the code, looks like:
* by passing `--generate-helper-explain-analyze-role new-role-name` with `--generate-helper-sql`, create an explain analyze role with new-role-name + generate a explain analyze helper function

I think instead of renaming `generate-helper-sql`, generating the explain analyze role can be completely separate thing. IOW, keep `generate-stats-helper-sql` to generate stats related helpers, then introduce a new separate flag something like `generate-explain-analyze-helper-sql` which will generate the helper SQL + create roles.
In that way, the docs will be more straight forward and saying "run collector with `--generate-explain-analyze-helper-sql myrole` then apply the SQL". Just an idea though.

Reply:
Yeah, that's a good point, let me revise it to just make it a completely separate helper - the intent here is different, and so maybe best to keep it split in two (and avoid the deprecation too).